src/backend/distributed/planner/query_pushdown_planning.c

```diff
@@ -53,6 +53,16 @@ typedef enum RecurringTuplesType
 	RECURRING_TUPLES_RESULT_FUNCTION
 } RecurringTuplesType;
 
+/*
+ * FlattenJoinVarsMutatorContext stores context variable needed by
+ * FlattenJoinVarsMutator function.
+ */
+typedef struct FlattenJoinVarsMutatorContext
+{
+	Query *queryTree;
+	PlannerInfo *plannerInfo;
```

Comment:
We don't use the planner info anymore. To simplify the code, can't we get rid of the context and only pass the `queryTree`?

Reply:
makes sense